cfpb/consumerfinance.gov
# Pass arguments to Cypress in our `yarn cy` script

This change allows us to pass Cypress arguments, such as `--spec` or `--config`, to Cypress via our `yarn cy` convenience script, rather than having to run Cypress directly with its config file location.


## How to test this PR

Try to pass some arguments to `yarn cy` before and after this PR. For example:

```
yarn cy --spec test/cypress/integration/consumer-tools/financial-well-being/financial-well-being.cy.js --config baseUrl=https://beta.consumerfinance.gov
```

Note that the behavior of `yarn cy` (to run the entire suite headless) remains the same if no arguments are passed.


## Checklist
